py3status/modules/static_string.py

```diff
@@ -3,35 +3,31 @@
 Display static text.
 
 Configuration parameters:
-    format: text that should be printed (default '')
+    format: default format for this module (default 'hello, world!')
```

Comment:
> default format for this module

this sounds wrong to me.  We don't really have a good wording some modules use `display format for this module` which is slightly better.